# P.syringae-leaf-colonization

Published (September 4, 2019):
https://www.pnas.org/content/early/2019/09/03/1908858116  
"Genome-wide identification of Pseudomonas syringae genes required for fitness during colonization of the leaf surface and apoplast"

Preprint (May 20, 2019):
https://biorxiv.org/content/10.1101/637496v1  
"Genome-wide identification of Pseudomonas syringae genes required for competitive fitness during colonization of the leaf surface and apoplast"  

The best way to view the data is in the Fitness Browser  
http://fit.genomics.lbl.gov/

This respository contains the additional data:  
all.poolcount (the counts of reads for each strain in each sample)

Experiments included:

Name	Description  
set1IT001 time0  
set1IT002 time0  
set1IT005	KB_1  
set4IT009 time0  
set4IT010 time0  
set4IT013	KB_2  
set1IT047 time0  
set1IT048 time0  
set1IT049	epiphytic_1a  
set1IT050	epiphytic_1b  
set2IT078 time0  
set2IT079 time0  
set2IT080	epiphytic_2a  
set2IT081	epiphytic_2b  
set3IT029 time0  
set3IT030 time0  
set3IT031	epiphytic_3a  
set3IT032	epiphytic_3b  
set2IT072 time0  
set2IT073 time0  
set2IT076	apoplastic_1a  
set2IT077	apoplastic_1b  
set2IT092 time0  
set2IT093 time0  
set2IT095	apoplastic_2  
set6IT081 time0  
set6IT082 time0  
set6IT083	apoplastic_3a  
set6IT084	apoplastic_3b  


With the exception of KB experiments and apoplastic_2 (set2IT095), all plant experiments are paired technical replicates sequenced separately.  
Sequencing replicates of time0 samples were combined for a given experiment.  
For the analysis in this paper, fitness values from technical replicates were calculated separately, and averaged for a given experiment. 
